docs/README.md

```diff
@@ -14,11 +14,7 @@ When pull requests are merged, the CI will upload the generated packages to the
   + [Developing Recipes Locally](developing_recipes_locally.md)
   + [Adding Packages to ConanCenter](adding_packages/README.md) :point_left: Best place to learn how to contribute
   + [Bumping versions to existing packages](bump_version.md)
-  + [Errors from the conan-center hook (KB-Hxxx)](error_knowledge_base.md)
   + [Review Process](review_process.md)
-  + [Labels](labels.md)
-  + [Supported platforms and configurations](supported_platforms_and_configurations.md)
```

Comment:
I would probably keep this and explicitly list the current platforms and also add plans for the immediate future (clang, newer versions of gcc and apple-clang, and Android)

Reply:
Addressed by https://github.com/conan-io/conan-center-index/pull/25875/commits/75461cc13016357ac2b7a51d7e2c5b060afa05af